modules/nw-routeadvertisements-setup.adoc

```diff
@@ -0,0 +1,484 @@
+Default local pref 100, local AS 64512
+Status codes:  s suppressed, d damped, h history, * valid, > best, = multipath,
+               i internal, r RIB-failure, S Stale, R Removed
+Nexthop codes: @NNN nexthop's vrf id, < announce-nh-self
+Origin codes:  i - IGP, e - EGP, ? - incomplete
+RPKI validation codes: V valid, I invalid, N Not found
+
+    Network          Next Hop            Metric LocPrf Weight Path
+ *>i10.128.0.0/23    192.168.111.22           0    100      0 i
+ *>i10.128.2.0/23    192.168.111.23           0    100      0 i
+ *>i10.129.0.0/23    192.168.111.20           0    100      0 i
+ *>i10.129.2.0/23    192.168.111.24           0    100      0 i
+ *>i10.130.0.0/23    192.168.111.21           0    100      0 i
+ *>i10.130.2.0/23    192.168.111.40           0    100      0 i
+ *>i10.131.0.0/23    192.168.111.25           0    100      0 i
+ *>i10.131.2.0/23    192.168.111.47           0    100      0 i
+ *> 192.168.1.0/24   0.0.0.0                  0         32768 i
+ *> 192.169.1.1/32   0.0.0.0                  0         32768 i
+----
+
+. Find the frr-k8s pod for each cluster node:
++
+[source,terminal]
+----
+$ oc -n openshift-frr-k8s get pod -owide
+NAME                                      READY   STATUS    RESTARTS   AGE   IP               NODE                                       NOMINATED NODE   READINESS GATES
+frr-k8s-86wmq                             6/6     Running   0          25h   192.168.111.20   master-0                                   <none>           <none>
+frr-k8s-h2wl6                             6/6     Running   0          25h   192.168.111.21   master-1                                   <none>           <none>
+frr-k8s-jlbgs                             6/6     Running   0          25h   192.168.111.40   node1.example.com   <none>           <none>
+frr-k8s-qc6l5                             6/6     Running   0          25h   192.168.111.25   worker-2                                   <none>           <none>
+frr-k8s-qtxdc                             6/6     Running   0          25h   192.168.111.47   node2.example.com   <none>           <none>
+frr-k8s-s5bxh                             6/6     Running   0          25h   192.168.111.24   worker-1                                   <none>           <none>
+frr-k8s-szgj9                             6/6     Running   0          25h   192.168.111.22   master-2                                   <none>           <none>
+frr-k8s-webhook-server-6cd8b8d769-kmctw   1/1     Running   0          25h   10.131.2.9       node3.example.com   <none>           <none>
+frr-k8s-zwmgh                             6/6     Running   0          25h   192.168.111.23   worker-0                                   <none>           <none>
+----
+
+. From the {product-title} cluster, check BGP routes on the cluster node's frr-k8s pod in the FRR container:
++
+[source,terminal]
+----
+$ oc -n openshift-frr-k8s -c frr rsh node1.example.com
```

Comment:
should use an actulal frr-k8s pod's name instead of "node1.example.com", for example:
$ oc -n openshift-frr-k8s -c frr rsh  frr-k8s-86wmq